test/unit/helpers.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-env mocha */
+
+var chai = require('chai')
+var sinon = require('sinon')
+var JsReporters = require('../../dist/js-reporters.js')
+var expect = chai.expect
+
+chai.use(require('sinon-chai'))
+
+describe('Helpers', function () {
+  var dummyFunc = function () {}
+
+  describe('autoregister', function () {
+    it('should register the QUnitAdapter', function () {
+      GLOBAL.QUnit = {
+        begin: sinon.stub(),
+        testStart: dummyFunc,
+        log: dummyFunc,
+        testDone: dummyFunc,
+        done: dummyFunc
+      }
+
+      JsReporters.autoRegister()
+
+      expect(GLOBAL.QUnit.begin).to.have.been.calledOnce
+
+      delete GLOBAL.QUnit
```

Comment:
Might be good to move this to an `afterEach`, to prevent this from leaking into another test.